doc/users/prev_whats_new/whats_new_2.1.0.rst

```diff
@@ -395,12 +395,22 @@ cases.
 ---------------------------------------------------
 
 Bulk setting of tick label rotation is now possible via
-:func:`~matplotlib.axis.Axis.set_tick_params` using the ``rotation``
+:func:`~matplotlib.axes.Axes.tick_params` using the ``rotation``
 keyword.
 
 ::
 
-    ax.xaxis.set_tick_params(which='both', rotation=90)
+    ax.tick_params(which='both', rotation=90)
+
+
+Ticklabels are turned off instead of being invisible
+----------------------------------------------------
+
+Internally, :func:`~matplotlib.axis.Axis.set_tick_params` is now used to
+hide tick labels instead of setting the visibility on the tick label objects.
+This improves overall performance and fixes some issues.
+As a consequence, in case those labels ought to be shown, `set_tick_params`
```

Comment:
Here also, `tick_params`.

Reply:
In the PR that changed that, they use Axis.set_tick_params

Reply:
I know, but saying "use `set_tick_params` as in `ax.tick_params(labelbottom=True)` is likely to make people think there is a typo somewhere.  What's *really* going on internally is that the Tick.label1On parameter is being set, and this can be done at any point in a chain of calls starting with ax.tick_params.

I think the problem is actually starting with the first line; the use of Axis.set_tick_params is irrelevant, and the point is not that tick labels are being hidden in a different way when subplots are shared, but that instead of being flagged as invisible, they are simply not drawn at all.  The user can turn label drawing on or off via, e.g., `ax.tick_params(labelbottom=True)`.